main.go

```diff
@@ -96,6 +96,8 @@ var (
 	restConfigBurst             int
 	webhookPort                 int
 	webhookCertDir              string
```

Comment:
Le'ts add them also to:
* test/extension/main.go
* test/infrastructure/docker/main.go
* test/infrastructure/inmemory/main.go

Reply:
done